core/src/test/scala/com/salesforce/op/stages/impl/classification/MultiClassificationModelSelectorTest.scala

```diff
@@ -311,9 +327,90 @@ class MultiClassificationModelSelectorTest extends FlatSpec with TestSparkContex
     val fitted = modelSelector.fit(data)
 
     fitted.modelStageIn.parent.extractParamMap().toSeq
-      .collect{ case p: ParamPair[_] if p.param.name == "impurity" => p.value }.head shouldBe myParam
+      .collect { case p: ParamPair[_] if p.param.name == "impurity" => p.value }.head shouldBe myParam
 
     val meta = ModelSelectorSummary.fromMetadata(fitted.getMetadata().getSummaryMetadata())
     meta.validationResults.head shouldBe myMetadata
   }
+
+  it should "trim low-cardinality labels during cross validation" in {
+    import org.scalatest.prop.TableDrivenPropertyChecks._
```

Comment:
you can simply mixin `org.scalatest.prop.PropertyChecks` into your test instead of local import